commands/stats.js

```diff
@@ -0,0 +1,77 @@
+import * as bungie from '../bungieApi';
+import * as util from 'util';
+
+export const command = '/stats';
+export async function handler(payload) {
+  // do some stuff with the payload
+
+  const name = payload.text.length ? payload.text : payload.user_name;
+  const msg = {
+    "response_type": "ephemeral",
+    attachments: []
+  };
+
+  const member = [
+    ...await bungie.getPlayerID(1, name),
+    ...await bungie.getPlayerID(2, name)
+  ];
+
+  if(!member.length) {
+    return({
+      "response_type": "ephemeral",
+      text: util.format("Sorry, Bungie did not know anything about `%s`", name)
```

Comment:
instead of util.format, better to just use the built in string interpolation:

```javascript
return ({
  response_type: 'ephemeral',
  text: `Sorry, Bungie did not know anything about '${name}'`,
});
```

You use backticks to quote a string that contains interpolation.